### PR TITLE
add-group-permissions-for-resources-fix

### PR DESCRIPTION
**Pull request by tuxing1986**
changes code for Topcoder - Add Group Permission Check For Adding Resources version 1.0

### CODE REVIEW

File: conf/MessageResources.properties [hunk 1]
```diff
@@ -1139,7 +1139,7 @@ editLateDeliverable.Response.label=Response:
 editLateDeliverable.ValidationFailed=There were validation errors. See below.
 
 #  -----  /jsp/editFinalReview.jsp page  -----
-viewScorecard.MinimumPassingScore=The minimum passing score:  
+viewScorecard.MinimumPassingScore=The minimum passing score\:
```
Comment (skyhit) [inline conf/MessageResources.properties]: @tuxing1986 why add `\`?

File: conf/OnlineReview.xml [hunk 2]
```diff
@@ -5213,6 +5213,22 @@
         <Property name="topcoder_api_base_url">
             <Value>@topcoder_api_base_url@</Value>
         </Property>
+        
+        <Property name="user_group_membership_url">
+            <Value>http://192.168.31.105:8084/v3/groups?memberId=%samp;membershipType=User-Group</Value>
```
Comment (skyhit) [inline conf/OnlineReview.xml]: @tuxing1986 this should be tokenized from token.properties, similar as `@topcoder_api_base_url@` above

File: conf/MessageResources.properties (hunk 1 above)
Comment (tuxing1986) [reply conf/MessageResources.properties]: The conf/MessageResources.properties  from the code base has "\".  I did not modify this configuration.

File: conf/OnlineReview.xml (hunk 2 above)
Comment (tuxing1986) [reply conf/OnlineReview.xml]: Should the token name configured by the admin?

Comment (tuxing1986) [reply conf/OnlineReview.xml]: The @topcoder_api_base_url@ in the token.properties.example is defined as 'http://api.topcoder.com/v2/develop/challenges'

**Pull request closed without merging**